docs/source/faq.rst

```diff
@@ -296,9 +302,9 @@ SEE ALSO: the :doc:`/using/envs` section of our documentation site.
 
 **Linux, OS X:** ``source deactivate myenv``
 
-**Windows:** deactivate myenv``
+**Windows:** ``deactivate myenv``
 
-NOTE: It is good practice to deactivate one environment before activating another.
+NOTE: In Windows it is good practice to deactivate one environment before activating another.
```

Comment:
Are there issues if you don't do this? It really shouldn't matter, but I know that activate on Windows has a lot of bugs in it.